aws/table_aws_ssm_document.go

```diff
@@ -0,0 +1,262 @@
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "name",
+				Description: "The name of the Systems Manager document.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "document_version",
+				Description: "The document version.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "status",
+				Description: "The user in your organization who created the document.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsSSMDocument,
+			},
+			{
+				Name:        "default_version",
+				Description: "The default version.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsSSMDocument,
+			},
+			{
+				Name:        "schema_version",
+				Description: "The schema version.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsSSMDocument,
+			},
+			{
+				Name:        "owner",
+				Description: "The AWS user account that created the document.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "version_name",
+				Description: "The version of the artifact associated with the document.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "document_format",
+				Description: "The document format, either JSON or YAML.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "document_type",
+				Description: "The type of document.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "platform_types",
+				Description: "The operating system platform.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "review_status",
+				Description: "The current status of the review.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "target_type",
+				Description: "The target type which defines the kinds of resources the document can run on.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "approved_version",
+				Description: "The version of the document currently approved for use in the organization.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsSSMDocument,
+			},
+			{
+				Name:        "attachments_information",
+				Description: "Details about the document attachments, including names, locations, sizes,and so on.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsSSMDocument,
+			},
+			{
+				Name:        "created_date",
+				Description: "The date when the document was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Hydrate:     getAwsSSMDocument,
+			},
+			{
+				Name:        "latest_version",
+				Description: "The latest version of the document.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getAwsSSMDocument,
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags associated with document",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Tags"),
+			},
+
```

Comment:
Can we add column for these properties?
```
	// The user in your organization who created the document.
	Author *string `type:"string"`

	// A description of the document.
	Description *string `type:"string"`

	// The Sha256 or Sha1 hash created by the system when the document was created.
	//
	// Sha1 hashes have been deprecated.
	Hash *string `type:"string"`

	// The hash type of the document. Valid values include Sha256 or Sha1.
	//
	// Sha1 hashes have been deprecated.
	HashType *string `type:"string" enum:"DocumentHashType"`

	// A description of the parameters for a document.
	Parameters []*DocumentParameter `type:"list"`

	// The version of the document that is currently under review.
	PendingReviewVersion *string `type:"string"`

	// A list of SSM documents required by a document. For example, an ApplicationConfiguration
	// document requires an ApplicationConfigurationSchema document.
	Requires []*DocumentRequires `min:"1" type:"list"`

	// Details about the review of a document.
	ReviewInformation []*ReviewInformation `min:"1" type:"list"`

	// The SHA1 hash of the document, which you can use for verification.
	Sha1 *string `type:"string"`

	// A message returned by AWS Systems Manager that explains the Status value.
	// For example, a Failed status might be explained by the StatusInformation
	// message, "The specified S3 bucket does not exist. Verify that the URL of
	// the S3 bucket is correct."
	StatusInformation *string `type:"string"`
```

Reply:
@anisadas did you ever add these columns in? did you encounter any issues trying to get these into the table?